### PR TITLE
Some new styling to home page

### PR DESCRIPTION
**Pull request by justindiaw**

### CODE REVIEW

File: C4SGSvc/logs/C4SGSvcLog.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[INFO ] 2017-02-16 18:52:26.066 [main] C4SgApplication - Starting C4SgApplication on WIN-IGTOHLTMPQT with PID 2164 (C:\Users\Administrator\Desktop\C4SG\C4SGSvc\target\classes started by Administrator in C:\Users\Administrator\Desktop\C4SG\C4SGSvc)
```
Comment (felipealves-gnu) [inline C4SGSvc/logs/C4SGSvcLog.log]: Don't commit log files.

**Comment by felipealves-gnu** [comment]
I'm getting all those errors when I try to compile your branch's branch like below.

screen shot 2017-02-26 at 01 57 27
screen shot 2017-02-26 at 01 57 45

**Comment by justindiaw** [comment]
@felipealves-gnu 
Thank you for your precious time!
The red color warnings are about the images. So is it because those images are missing from your computer?
And the yellow warnings are almost all about the formatting problem and , in addition, I should delete private for that function.
Anyway, those errors have not shown up when compiling in my computer. What should I do now? Should I close the pull request and solve the errors and try to pull again?

**Comment by felipealves-gnu** [comment]
@justindiaw the red errors are errors related to the path, change the first path error to ./images/logo-transparent.png and check if it works in your machine.

Second keep the PR opened, just clean those warnings and try change those paths, commit (with a refactor message) and push again using the same branch as you did before so it will automatically be pushed to the same PR.

**Pull request merged**